roylines/node-epimetheus
# fix request.path() && res.charSet() is not a function

## What does this PR do
This PR fixes `request.path()` by `request.path`
The latter was causing a request.path() is not a function error

## How can it be tested
Cd into  _examples/express/_  folder
and run `node express`
Then in your browser go to `localhost:8000/metric`
You should see the metrics.
